portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase

```diff
@@ -89,4 +92,275 @@ definition {
+				value1 = "test1");
+
+			Button.clickSave();
+		}
+
+		task ("Assert that the user does not have permission to access the admin page") {
+			AssertTextEquals(
+				locator1 = "Portlet#ERROR",
+				value1 = "You do not have the roles required to access this portlet.");
+		}
+	}
+
+	@description = "Bug: LRAC-11339 | Automation ID: LRAC-11365 | Test Summary: A new user created in the admin part can be invited to the workspace and see the data, the same user now with admin permission also can see the data"
+	@priority = "5"
+	test NewUserIndependentOfRoleCanSeeDataInTheWorkspace {
+		var userFirstName = "ac";
+		var userLastName = "ac";
+		var userScreenName = "ac";
+		var userEmailAddress = "ac@liferay.com";
+		var currentURL = Navigator.getCurrentURL();
+
+		task ("Connect AC and DXP and generate some events") {
+			ACUtils.launchAC();
+
+			ACSettings.takeTokenFromAnalyticsCloud();
+
+			ACUtils.launchDXP();
+
+			ACUtils.addSiteAndPage();
+
+			ACUtils.createFormAndAddToPage();
+
+			ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+			ACUtils.navigateToSitePage(
+				actionType = "Submit Form",
+				pageName = "AC Page",
+				siteName = "Site Name",
+				submitText = "Testing");
+
+			ACUtils.closeAllSessionsAndWait();
+		}
+
+		task ("Navigate to the AC Admin page") {
+			ACUtils.launchAC();
+
+			Navigator.openSpecificURL(url = "${currentURL}admin");
+		}
+
+		task ("Go to control panel > Users > Users and Organizations") {
+			ProductMenuHelper.expandPanel(panel = "control panel");
+
+			ProductMenuHelper.expandCategory(category = "Users");
+
+			ProductMenuHelper.gotoPortlet(portlet = "Users and Organizations");
+		}
+
+		task ("Create a new user") {
+			LexiconEntry.gotoAdd();
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#DETAILS_SCREEN_NAME_FIELD",
+				value1 = "${userScreenName}");
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#DETAILS_EMAIL_ADDRESS_FIELD",
+				value1 = "${userEmailAddress}");
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#DETAILS_FIRST_NAME_FIELD",
+				value1 = "${userFirstName}");
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#DETAILS_LAST_NAME_FIELD",
+				value1 = "${userLastName}");
+
+			AssertClick(
+				locator1 = "Button#SUBMIT",
+				value1 = "Save");
+
+			UserNavigator.gotoUser(userScreenName = "${userScreenName}");
+
+			UserNavigator.gotoPassword();
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#PASSWORD_NEW_PASSWORD_FIELD",
+				value1 = "test");
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#PASSWORD_ENTER_AGAIN_FIELD",
+				value1 = "test");
+
+			Button.clickSave();
+		}
+
+		task ("Go to AC with Test Test > Settings > User Management > Invite the new user to the workspace") {
+			ACUtils.launchAC();
+
+			ACNavigation.goToSettings();
+
+			ACSettings.goToUserManagement();
+
+			ACSettings.addUser(userInvite = "ac@liferay.com");
+
+			ACUtils.searchBar(searchItem = "ac@liferay.com");
```

Comment:
Hey @manuferreira, I think this search here is not necessary. What do you think?